### DISCUSSION TITLE
CHILD_SA status incase SPD policies fails and IKE_SA success

### DISCUSSION THREAD

**Discussion by itsme2304** [strongswan/strongswan#1479] [Q&A]
Hi All,

Please, let me know if there is any command or way to check whether CHILD_SA (phase 2) has succeeded.
In my case, phase-1 IKE_SA is succeeded and phase-2 CHILD_SA fails.

any help or pointers are appreciated. 

logs snippets.
	Nov 10 13:51:16  ipsec: 16[KNL] policy no interface 17.17.132.113
	Nov 10 13:51:16  ipsec: 16[IKE] unable to install IPsec policies (SPD) in kernel
	Nov 10 13:51:16  ipsec: 16[IKE] failed to establish CHILD_SA, keeping IKE_SA
	Nov 10 13:51:16  ipsec: 16[IKE] sending DELETE for ESP CHILD_SA with SPI cd3c359b

**Comment by tobiasbrunner** [comment]
> Please, let me know if there is any command or way to check whether CHILD_SA (phase 2) has succeeded.

What do you mean?

> Nov 10 13:51:16 ipsec: 16[KNL] policy no interface 17.17.132.113

Hm, no idea where that message comes from. Have you modified the code in any way? What kernel interface do you use?
